pyxem/signals/insitu_diffraction2d.py

```diff
@@ -55,12 +55,11 @@ def roll_time_axis(self, time_axis):
         return self.rollaxis(time_axis, 2)
 
     def get_time_series(self, roi=None, time_axis=2):
-        """
-        Create a intensity time series from virtual aperture defined by roi.
+        """Create a intensity time series from virtual aperture defined by roi.
 
         Parameters
         ----------
-        roi: :obj:`hyperspy.roi.BaseInteractiveROI`
+        roi: :obj:`~hyperspy.roi.BaseInteractiveROI`
```

Comment:
I don't think we need the reST Sphinx syntax here?

```suggestion
        roi : hyperspy.roi.BaseInteractiveROI
```

Reply:
I kind of like the explicit references.  I think if the class isn't imported it seems like the Sphinx mapping doesn't work At least I think that is the case.  Maybe if we add in type checking we could simplify things but I'd rather have things consistent for now.